support/cas-server-support-cassandra-ticket-registry/src/test/resources/logback.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file needs to entirely go away. No logback is allowed as it messes up the rest of the modules working with and using log4j.